pkg/controllers/status/work_status_controller.go

```diff
@@ -82,7 +82,7 @@ type WorkStatusController struct {
 // The Controller will requeue the Request to be processed again if an error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (c *WorkStatusController) Reconcile(ctx context.Context, req controllerruntime.Request) (controllerruntime.Result, error) {
-	klog.V(4).Infof("Reconciling status of Work %s.", req.NamespacedName.String())
+	klog.V(4).InfoS("Reconciling status of Work.", "work", req.NamespacedName.String())
```

Comment:
```suggestion
	klog.V(4).InfoS("Reconciling status of Work.", "namespace", req.Namespace, "name", req.Name)
```
I guess what @jabellard expected is like this, logging `namespace` and `name` separately. 

Same as below.

Reply:
Thank you @RainbowMango!

To confirm, you would like all `work` fields currently of the form `<namespace>/<resource>` split out to `namespace` and `name` fields?

If so, I'll update lines:
* [119](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR119)
* [131](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR131)
* [175](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR175)
* [219](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR219)
* [232](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR232)
* [353](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR353)
* [519](https://github.com/karmada-io/karmada/pull/6578/files#diff-c628439b001f1de5f834207e20b1b28d6d8c4d7d40993d31a9873916bc786d5aR519)